.github/workflows/ci.yml

```diff
@@ -0,0 +1,275 @@
+name: CI
```

Comment:
This is a leftover of the merge, it has been split into L1 and L2 CIs, the paths are outdated, so please remove this file